.ci-orchestrator/scheduledPersonalBuilds.yml

```diff
@@ -0,0 +1,240 @@
+type: pipeline_definition
+product: OpenLiberty
+name: OpenLiberty Personal Build
+description: A build run against OpenLiberty Pull Requests
+triggers:
+- type: schedule
+  triggerName: "Personal product only change"
+  triggerRank: 50
+  triggerMonitored: false
+  triggerThreshold: 100
+  every:
+    hours: 24
+  propertyDefinitions:
+  - name: git.laos.clone.repository.username
+    defaultValue: "fritze2"
+    steps:
+    - stepName: compile
+    - stepName: unittest
+  - name: git.laos.clone.checkout.branch
+    defaultValue: "test-change-com.ibm.ws.jca""
```

Comment:
Double quote here might be erroneous.

Reply:
Good spot, it is. I've remove the double quote.